### DISCUSSION TITLE
GPU is too old

### DISCUSSION THREAD

**Discussion by damon2410** [openai/whisper#1165] [Q&A] [answered]
Hi everyone, I got problem in too old VGA. "Found GPU0 Quadro K2000 which is of cuda capability 3.0. PyTorch no longer supports this GPU because it is too old. The minimum cuda capability supported by this library is 3.7". How can I fix them? Please!

**Comment by jongwook** [comment, marked as answer]
Unfortunately [PyTorch 0.3.0 was the last version that supports CUDA compute capability 3.0](https://github.com/pytorch/pytorch/releases/tag/v0.3.1), and it's very unlikely that PyTorch 0.3.0 is compatible with the code in this repository. K2000 is ~10 years old at this point and would probably perform no better than running on your CPU even if it is compatible.

You can also try [whisper.cpp](https://github.com/ggerganov/whisper.cpp) which performs better on CPU.
